docs/resources/consul_cluster.md

```diff
@@ -26,15 +30,14 @@ resource "hcp_consul_cluster" "example" {
 }
 ```
 
--> **Note:** The `primary_link` attribute is related to federation, a feature that is currently in private beta.
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
 ### Required
 
 - **cluster_id** (String) The ID of the HCP Consul cluster.
 - **hvn_id** (String) The ID of the HVN this HCP Consul cluster is associated to.
-- **tier** (String) The tier that the HCP Consul cluster will be provisioned as.  Only `development` and `standard` are available at this time.
+- **tier** (String) The tier that the HCP Consul cluster will be provisioned as.  Only `development`, `standard` and `plus` are available at this time.
```

Comment:
I think you might need to regen the docs to include the added link to the pricing page.